tck/src/main/java/org/eclipse/microprofile/lra/tck/TckRecoveryTests.java

```diff
@@ -56,6 +56,8 @@
  */
 @RunWith(Arquillian.class)
 public class TckRecoveryTests {
+    
+    public static final String DEPLOYMENT_URL_HEADER = "Deployment-URL";
```

Comment:
Why not prefix the header name with `LRA` or `LRA_TCK` to avoid conflicting with any existing headers or to easily see that it is one of ours.